tasks/ubuntu/bionic.task/store_ip.erb

```diff
@@ -0,0 +1,12 @@
+<%# -*- shell-script -*- %>
+# Get current IP
+node_ip=$(ip addr show | grep -A 2 BROADCAST | grep inet | awk '{ print $2 }' | awk -F'/' '{print $1}')
+echo IP is $node_ip
+
+echo "$node_ip" > /tmp/store_ip.txt
+
+# Send IP up
+curl -s <%= store_url("ip" => "$node_ip") %>
+
+# Send final state
+curl -s <%= stage_done_url("finished") %>
```

Comment:
This looks like a duplicate of `ubuntu.task/os_boot.erb`. Why move it into the storing of IP?

Reply:
this stage the last one, by adding this I got all the stages to run and the nodes get registered.

Reply:
I would expect a `"finished"` log message to be reported to the server already, via this in `os_boot.erb` (which renders this file):
```
<%= render_template("store_ip") %>

<%= render_template("os_complete") %>

# Send final state
curl -s <%= stage_done_url("finished") %>
```
Is that last `"finished"` message not showing up?

Reply:
my mistake, i removed this.